onnxruntime/test/python/transformers/test_gqa_cpu.py

```diff
@@ -1773,9 +1756,11 @@ def parity_check_gqa_past_no_buff(
         numpy.mean(numpy.abs(out - out_ref)),
         correct,
     )
+    return all_close
 
 
 class TestGQA(unittest.TestCase):
+    @pytest.mark.slow
```

Comment:
Could we add some test case that is not marked slow? For example, one case for prompt, and one for token generation.